cfr-pentaho5/resources/cfr.properties

```diff
@@ -0,0 +1,3 @@
+#upload.security.matchers=ext,wht,blk
```

Comment:
Documentation on how to use these properties ? Both in the properties file itself as well as on the readme.md for the project